### PR TITLE
[CAPPL] Fixes to registry syncer

### PR DESCRIPTION
**Pull request by cedric-cordenier**

### CODE REVIEW

File: core/services/workflows/syncer/workflow_registry.go [hunk 1]
```diff
@@ -265,14 +285,14 @@ func (w *workflowRegistry) handlerLoop(ctx context.Context) {
 			}
 
 			if resp.Err != nil || resp.Event == nil {
-				w.lggr.Errorf("failed to handle event: %+v", resp.Err)
+				w.lggr.Errorf("failed to handle event", "err", resp.Err)
```
Comment (george-dorin) [inline core/services/workflows/syncer/workflow_registry.go]: Should this be `Errorw`?

Comment (cedric-cordenier) [reply core/services/workflows/syncer/workflow_registry.go]: It should :) Fixed, thank you

File: core/capabilities/triggers/logevent/trigger.go [hunk 2]
```diff
@@ -67,6 +67,11 @@ func newLogEventTrigger(ctx context.Context,
 		return nil, nil, err
 	}
 
+	err = contractReader.Start(ctx)
```
Comment (ettec) [inline core/capabilities/triggers/logevent/trigger.go:70]: can you expand why this change was needed in this PR?

**Review by ettec — APPROVED** [review]

**Review by agparadiso — DISMISSED** [review]

File: core/capabilities/triggers/logevent/trigger.go (hunk 2 above)
Comment (cedric-cordenier) [reply core/capabilities/triggers/logevent/trigger.go:70]: Yes -- previously this was being done in the testharness which is shared by the logevent trigger and by the workflow syncer. This caused the integration tests to pass for both even though any passed in contract readers would actually not work (because they would never be started.)

The fix is to cause the test to fail by not starting the contract reader in the test -- that happens here: https://github.com/smartcontractkit/chainlink/pull/15567/files#diff-23e2c7f1307a0cfa9724f11c5b20ffab5082fee238bffb021f54f605499fa427R121

And then to fix both things using the tests by starting the contract reader explicitly.

**Review by ettec — DISMISSED** [review]

**Review by george-dorin — DISMISSED** [review]

**Review by ettec — APPROVED** [review]

**Review by agparadiso — APPROVED** [review]

**Review by george-dorin — APPROVED** [review]
